app/commands/track/update_build_status.rb

```diff
@@ -274,21 +274,33 @@ def practice_exercises
       num_exercises: active_practice_exercises.size,
       num_exercises_target: practice_exercises_num_exercises_target,
       created: active_practice_exercises.map { |exercise| serialize_exercise(exercise) },
+      num_unimplemented: num_unimplemented_practice_exercises,
+      unimplemented: unimplemented_practice_exercises.map { |exercise| serialize_prob_specs_exercise(exercise) },
```

Comment:
I feel like `num_unimplemented` could just be calculated client-side using `unimplented.count`. If that's the case, then I think we need to rename the keys.

Reply:
It could. It is a pattern that's being used in other places too. We can change it, but I'd like to change it elsewhere too then

Reply:
Let me know if you'd prefer that